client/src/components/Conversation/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import './style.css';
+
+class Conversation extends Component {
+  render() {
+    return (
+
+      <div className="conversation-container">
+        <div className="conversation-container__email-container">
+          <p> Email : </p>
+          <p className="conversation-container__email-container_email">{this.props.reply.email}</p>
```

Comment:
there should not be 2 __ used in a classname. All elements are related to the same block, in this case, the ```conversation-container```.

the class name should be ```conversation-container__email```